backend/.env.example

```diff
@@ -1,32 +1,44 @@
-#
+#####################################
 # APPLICATION
-#
+#####################################
+
 NODE_ENV=development
 PORT=3001
 HOST=0.0.0.0
 FRONTEND_URL=http://localhost:3000
-SENTRY_DSN='your sentry dsn'
-#
+
+#####################################
 # AUTHENTICATION
-SECRET_KEY=secret
+#####################################
 
-#
+JWT_SECRET=secret
+
+#####################################
 # DATABASE CONNECTION SECRECTS
-#
+#####################################
+
 MYSQL_USER=autoline_user
 MYSQL_DATABASE=autoline
 MYSQL_ROOT_PASSWORD=secret
 MYSQL_PASSWORD=secret
 
+#####################################
 # Use these for the Docker development
+#####################################
+
 MYSQL_HOST=autoline_mariadb
 MYSQL_PORT=3306
 
-# Use these for the local development
-#MYSQL_HOST=localhost
-#MYSQL_PORT=3307
```

Comment:
These variables **mustn't** be removed! From the outer world, we can connect only to the exposed port, which is 3307 and `localhost`. `autoline_mariadb` and `3306` exists only in the Docker-Compose environment and cannot be addressed